src/data/finset/lattice.lean

```diff
@@ -991,6 +991,9 @@ begin
     refl }
 end
```

Comment:
I think this lemma should be renamed to `count_finset_sup` to avoid the prime below